test/test-e2e.sh

```diff
@@ -6,7 +6,7 @@ set -e
 : ${SKAFFOLD_PROFILE:-""}
 
 
-KIND_VERSION=0.14.0
+KIND_VERSION=0.15.0
```

Comment:
we don't really need to update that in the release branch since it is used for testing only

Reply:
done